examples/python/upcloud-server/Pipfile

```diff
@@ -0,0 +1,10 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[requires]
+python_version = "3"
+
+[packages]
+cdktf = {path = "./../../../dist/python/cdktf-0.0.0-py3-none-any.whl"}
```

Comment:
I think this changed in the mean time and we need to remove it

Reply:
I mean apparently it works, but removed it from the others as well

Reply:
Ah, right!

Reply:
Fixed it anyway 👍